Makefile

```diff
@@ -78,10 +78,10 @@ image: | $(BINDIR) ## build docker image targeting all supported platforms
 
 .PHONY: kind-load
 kind-load: local-images | $(BINDIR)/kind
-	$(BINDIR)/kind load docker-image $(CONTAINER_REGISTRY)/trust-manager:$(RELEASE_VERSION) $(CONTAINER_REGISTRY)/cert-manager-bundle-debian:latest
+	$(BINDIR)/kind load docker-image $(CONTAINER_REGISTRY)/trust-manager:$(RELEASE_VERSION) $(CONTAINER_REGISTRY)/cert-manager-package-debian:latest$(DEBIAN_TRUST_PACKAGE_SUFFIX)
```

Comment:
note: Adding `$(DEBIAN_TRUST_PACKAGE_SUFFIX)` to the end here was something I'd missed previously so I snuck it in here.